### PR TITLE
LL-238

### PR DESCRIPTION
**Pull request by KevinLemon112**
# Description

Ticket creates the ActiveMQ management service using the files in the src/notification folder of the LL repo. 

Subsequent # (issue)

LL-238

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

The way I verified my changes worked was to create a queue, send messages to the queue in the ActiveMQ management hub, and get the messages from there and make sure they sent. The steps are as follows:

1. Open VSCode and Docker Desktop

2. Be in the LL-238 directory (or the master if the ticket is already pushed) and then cd into the src/notification directory where are the files are located

3. Run

```shell
docker compose up --build
```

4. Go to http://localhost:4000/publish/3
5. Should see `{"id":"3","message":"From Publish Service"}`

# Checklist (when relevant):

- [x] My code follows the style guidelines of this project

    [TypeScript](https://google.github.io/styleguide/tsguide.html)
[Python](https://peps.python.org/pep-0008/)
[Schema](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/forefront-2010/ee652242(v=vs.100))
- [x] I have included a reviewer on the pull request [Reviewer 1 = @pogi7, Reviewer 2 = @erikajledesma ]
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pogi7 — CHANGES_REQUESTED** [review]
@KevinLemon112 

I'm getting the following error:
```typescript
 ~/gi/p/server/sr/notification  LL-238 <4 *2 !1 ?1  docker compose up -d           INT  41s  09:14:24 AM 
[+] Building 0.0s (0/0)                                                                     docker:colima
[+] Running 1/1
 ✔ Container rabbitmq  Started                                                                       0.0s 

 ~/gi/p/server/sr/notification  LL-238 <4 *2 !1 ?1  node connection.js                   ok  09:14:27 AM 
Error connecting to RabbitMQ: Error: connect ECONNREFUSED 127.0.0.1:5672
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 5672
}

File: src/notification/publish/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "publish",
+  "version": "1.0.0",
+  "description": "",
+  "main": "publish.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "publish": "npx ts-node publish.ts"
```
Comment (pogi7) [inline src/notification/publish/package.json:8]: @KevinLemon112 The reason why the publish notification service wasn't working previously was there was no `npx ts-node` that is required to compile and run typescript code.

File: src/notification/subscribe/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "subscribe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "subscribe.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "subscribe": "npx ts-node subscribe.ts"
```
Comment (pogi7) [inline src/notification/subscribe/package.json:8]: @KevinLemon112 The reason why the subscribe notification service wasn't working previously was there was no `npx ts-node` that is required to compile and run typescript code.

**Review by pogi7 — APPROVED** [review]
i'm approving, but @KevinLemon112 make a new ticket to fix the unit test that's not working

**Review by erikajledesma — APPROVED** [review]
looks goodd

**Pull request merged**